utils/src/main/scala/com/salesforce/op/utils/io/file/FileCompress.scala

```diff
@@ -0,0 +1,60 @@
+package com.salesforce.op.utils.io.file
+
+import java.io.{BufferedInputStream, FileInputStream, FileOutputStream}
+import java.nio.file.{Files, Paths}
+import java.util.zip.{ZipEntry, ZipFile, ZipOutputStream}
+
+import org.apache.commons.io.FileUtils
+
+import scala.collection.JavaConverters._
+
+case object FileCompress {
+
+  /**
+   * Zip a directory
+   * @param inLoc location of directory to zip
+   * @param zipLoc output zip file name
+   */
+  def zip(inLoc: String, zipLoc: String): Unit = {
```

Comment:
It could be beneficial to use [this](https://github.com/zeroturnaround/zt-zip) library instead. E.g
```scala
// Compress folder
ZipUtil.pack(new File("/tmp/demo"), new File("/tmp/demo.zip"))

// Decompress zip into a folder
ZipUtil.unpack(new File("/tmp/demo.zip"), new File("/tmp/demo"))
```

Reply:
The downsides are 
- it's yet another dependency to be vetted at Salesforce 
- and we lost the API that would allow to write the zip directly to the destination file system by providing an appropriate output stream [here](https://github.com/salesforce/TransmogrifAI/pull/516/commits/2b0ecd031294b9c699049b76356997e8bc85384d#diff-4fc4382e0d7736f51ed2d263d373ece5L22)

Reply:
Lets chat about this @gerashegalov

Reply:
`java.util.zip` could be an alternative. zt-zip seems pretty light-weight, though.